src/api/orders/orders.request.ts

```diff
@@ -0,0 +1,23 @@
+import {ActiveOrdersRequestParams} from './types'
+
+export class ActiveOrdersRequest {
+    public readonly page: number | undefined
+
+    public readonly limit: number | undefined
+
+    constructor(params: ActiveOrdersRequestParams) {
+        this.page = params.page
+        this.limit = params.limit
+    }
+
+    static new(params: ActiveOrdersRequestParams): ActiveOrdersRequest {
+        return new ActiveOrdersRequest(params)
+    }
```

Comment:
1. Make `params` optional
2. `return new ActiveOrdersRequest(params || {})`

Reply:
did it in a little bit other way

```
    constructor(params: ActiveOrdersRequestParams = {}) {
        this.page = params.page
        this.limit = params.limit
    }

    static new(params?: ActiveOrdersRequestParams): ActiveOrdersRequest {
        return new ActiveOrdersRequest(params)
    }
```

is it ok?

Reply:
yes